index.html

```diff
@@ -118,7 +123,12 @@ <h4>Document-Level Defaults</h4>
 			  
 			  <p>For specifications that can make use of the [[JSON-LD]] <code>@context</code> mechanism, use the <code>@language</code> and <code>@direction</code> fields to supply the document level defaults.</p>
 			  
-			  <p>If your specification must define its own document level defaults, provide two optional fields. The document's default language field should be called <code>language</code> and can contain a valid [[BCP47]] language tag. The document's default <a>base direction</a> field should be called <code>direction</code> and support the values <code>ltr</code>, <code>rtl</code>, or <code>auto</code>.</p>
+			  <p>If your specification defines its own document level defaults, provide two optional fields:</p>
+			  
+			  <ul>
+				  <li>The document's default language field SHOULD be called <code>language</code> and SHOULD be specified to contain a valid [[BCP47]] language tag. Specifications SHOULD specify that implementations are only require to check if a [[BCP47]] language tag is "well-formed".</li>
```

Comment:
Maybe we can add links to https://www.w3.org/TR/i18n-glossary/#dfn-valid and https://www.w3.org/TR/i18n-glossary/#dfn-well-formed here?